services/github/github-code-size.service.js

```diff
@@ -31,15 +31,8 @@ export default class GithubCodeSize extends BaseGithubLanguage {
 
   static defaultBadgeData = { label: 'code size' }
 
-  static render({ size }) {
-    return {
-      message: prettyBytes(size),
-      color: 'blue',
-    }
-  }
-
   async handle({ user, repo }) {
     const data = await this.fetch({ user, repo })
-    return this.constructor.render({ size: this.getTotalSize(data) })
+    return renderSizeBadge(this.getTotalSize(data), 'iec', 'code size')
```

Comment:
This number is never presented anywhere in the GitHub UI. Since the other 2 GitHub badges use IEC bytes, lets go with that here too.